README.md

```diff
@@ -1,146 +1,164 @@
 
-If want to resize a volumes after its initialization you need to make sure that it's supported by its underlying volume plugin:
-https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims
+## Installation
+> The following instructions on this guide are for local machines which runs on Linux operating systems. You can still use the same instructions with small to no changes on a MacOS device as well.
 
-If it's supported then it should be an easy process like this:
-https://www.jeffgeerling.com/blog/2019/expanding-k8s-pvs-eks-on-aws
+### Prepare
+1. Clone the repository to your local machine by using following command.
+It will download the dependent helm charts from Confluent platform for components such as Kafka and Zookeeper.
+    ```shell
+    git clone --recurse-submodules https://github.com/RADAR-base/RADAR-Kubernetes.git
+    ```
+ 
+2. Create basic config files using template files.
+    ```shell script
+    cd RADAR-Kubernetes
+    cp environments.yaml.tmpl environments.yaml
+    cp base.yaml production.yaml
+    cp base.yaml.gotmpl production.yaml.gotmpl
+    ```
+   
+     It is recommended make a private clone of this repository, if you want to version control your configurations and/or share with other people. 
+      
+     The best practise to share your platform configurations is by **sharing the encrypted version of `production.yaml`**.  
 
-## Authentication for monitoring
-If you want to have authentication you need to generate it somehow and there are couple of ways to do that. You can use these websites:
+### Configure
 
-http://www.htaccesstools.com/htpasswd-generator/
+#### Project Structure
+- [/bin](bin): Contains initialization scripts
+- [/charts](charts): Contains Helm Charts for RADAR-base components
+- [/cp-helm-charts](cp-helm-charts): Path where Helm charts from Confluent platform will be downloaded using git
+- [/etc](etc): Contains Helm charts for third party applications used in RADAR-base
+- [/helmfile.d](helmfile.d): Contains Helmfiles for modular deployment of the platform
+- [environment.yaml](environments.yaml): Defines current environment files in order to be used by helmfile. Read more about `bases` [here](https://github.com/roboll/helmfile).
+- [production.yaml](production.yaml): Production helmfile template to configure and install RADAR-base component. Inspect the file to enable, disable and configure components required for your usecase. The default helmfile enables all core components that are needed to run RADAR-base platform with pRMT and aRMT apps. If you're not sure which components you want to enable you can refer to wiki for [an overview and breakdown on RADAR-Base components and their roles](https://radar-base.atlassian.net/wiki/spaces/RAD/pages/2673967112/Component+overview+and+breakdown). 
```

Comment:
`environment.yml`, `production.yml` these links will always not work as there is only the stub file in the repo

Reply:
True. That's why I put the project structure section after prepare. I can remove the link as well